sql-plugin/src/main/scala/com/nvidia/spark/rapids/python/PythonConfEntries.scala

```diff
@@ -64,6 +64,13 @@ object PythonConfEntries {
     .booleanConf
     .createOptional
 
+  val PYTHON_ARENA_MEM = conf("spark.rapids.python.memory.gpu.arena.enabled")
```

Comment:
I think we need a follow on issue to actually enable this on the cudf python side.  I would almost rather have us not include this config until we have it working all the way through.

Reply:
Removed.